<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartFBClass</class>
 <widget class="QMainWindow" name="StartFBClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartFB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>14</y>
      <width>285</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="TitleLable">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Flappy Bird&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="username_label">
             <property name="text">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="UsernamelineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="password_label">
             <property name="text">
              <string>密码   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PasswordlineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="LoginpushButton">
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ClearpushButton">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="introLable">
         <property name="font">
          <font>
           <family>Ark Pixel 12px monospaced zh_cn</family>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;你好，欢迎来到&lt;span style=&quot; font-weight:700;&quot;&gt;Flappy Bird&lt;/span&gt;！&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Flappy Bird 的游戏玩法非常简单，&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;通过点击屏幕，使小鸟一直飞并穿过水管的空隙。🐦&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;如果你没有账号，&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;你可以在下拉游戏菜单选择以游客模式开始游戏&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;或者下拉用户菜单选择注册建立新帐号。&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;祝你玩得愉快！🎉&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_G">
    <property name="title">
     <string>游戏（&amp;G）</string>
    </property>
    <addaction name="action_V"/>
    <addaction name="action_R"/>
   </widget>
   <widget class="QMenu" name="menu_U">
    <property name="title">
     <string>用户（&amp;U）</string>
    </property>
    <addaction name="action_S"/>
    <addaction name="action_I"/>
    <addaction name="action_E"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>其他（&amp;O）</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_M"/>
   </widget>
   <addaction name="menu_G"/>
   <addaction name="menu_U"/>
   <addaction name="menu_O"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_S">
   <property name="text">
    <string>注册（&amp;S）</string>
   </property>
  </action>
  <action name="action_I">
   <property name="text">
    <string>导入用户信息（&amp;I）</string>
   </property>
  </action>
  <action name="action_E">
   <property name="text">
    <string>导出用户信息（&amp;E）</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>以游客模式开始游戏（&amp;V）</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>查看排行榜（&amp;R）</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于（&amp;A）</string>
   </property>
  </action>
  <action name="action_M">
   <property name="text">
    <string>指南（&amp;M）</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="StartFB.qrc"/>
 </resources>
 <connections/>
</ui>
